lib/guardian/token/verify.ex

```diff
@@ -62,20 +62,41 @@ defmodule Guardian.Token.Verify do
         ) :: {:ok, Guardian.Token.claims()} | {:error, any}
   @doc """
   For claims, check the values against the values found in
-  `claims_to_check`. If there is a claim to check that does not match
-  verification fails.
+  `claims_to_check`. If there is a claim to check that does not pass
+  verification, it fails.
+
+  When the value of a claim is a list, it checks that all values of
+  the same claim in `claims_to_check` are members of the list.
   """
   def verify_literal_claims(claims, nil, _opts), do: {:ok, claims}
 
   def verify_literal_claims(claims, claims_to_check, _opts) do
-    results = for {k, v} <- claims_to_check, into: [], do: verify_literal_claim(claims, k, v)
+    results =
+      for {k, v} <- claims_to_check, into: [] do
```

Comment:
Never used `for` on Elixir after 2 years 😄 

Can we use `map` or `reduce` or something similar like that instead of `for`?